playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -136,6 +136,31 @@ export default class ParticipantListPage {
     return download;
   }
 
+/**
+ * Filters the participant list to search for a specific participant when given their guid
+ * @param participantGUID the guid of the specific participant to search for
+ */
+public async filterListByParticipantGUID(participantGUID: string): Promise<void> {
+  const customizeViewPanel = this.filters.customizeViewPanel;
+  await customizeViewPanel.open();
+  await customizeViewPanel.selectColumns('Participant Columns', ['Participant ID']);
+
+  const searchPanel = this.filters.searchPanel;
+  await searchPanel.open();
+  await searchPanel.text('Participant ID', {textValue: participantGUID });
+  await searchPanel.search();
+}
+
+/**
+ * Selects a specific participant from the participant list when given their guid
+ * @param participantGUID the guid of the specific participant to search for
+ */
+public async selectParticipant(participantGUID: string): Promise<void> {
+  await this.page.getByRole('cell', { name: participantGUID }).click()
```

Comment:
Use [openParticipantPageAt(position: number)](https://github.com/broadinstitute/ddp-angular/blob/03a9f2eed954ecd8f99cdf924be8fb6c2807977b/playwright-e2e/dsm/component/tables/participant-list-table.ts#L31)
I don't think you need to write this func.